src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.binding
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
+  val psTaskWdl = """
+    |task ps {
+    |  command {
+    |    ps
+    |  }
+    |  output {
+    |    File procs = stdout()
+    |  }
+    |}""".stripMargin
+
+  val cgrepTaskWdl = """
+    |task cgrep {
+    |  command {
+    |    grep '${pattern}' ${File in_file} | wc -l
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val wcTaskWdl = """
+    |task wc {
+    |  command {
+    |    wc -l ${File in_file}
```

Comment:
Personally I'd probably prefer a hard-line approach to catch more fundamental screw-ups; it's easy enough to pipe something that produces output like the above to `awk {print $1}` if only the first number is desired.  Feel free to disagree... :smile: